src/main/resources/init_db.sql

```diff
@@ -18,3 +18,25 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
```

Comment:
maybe better add "DROP TABLE IF EXISTS 'cars';" as with "manufacturers" and "drivers" in this file